Istio migration docs (#4246)
kyma-project/kyma: docs/application-connector/08-10-rotate-root-ca.md

  kubectl -n kyma-integration edit secret connector-service-app-ca
  ```

5. Get the existing Istio CA certificate. Fetch it from the `application-connector-ca-certs` Secret and save it to a `old-ca.crt` file. Run:
  ```
  kubectl -n istio-system get secret application-connector-ca-certs -o=jsonpath='{.data.ca\.crt}' | base64 --decode > old-ca.crt
  ```

6. Merge the old Nginx certificate and the newly generated certificate into a single `nginx-ca.crt` file:
  ```
  cat old-ca.crt new-ca.crt > merged-ca.crt
  ```

7. Encode the newly created `merged-ca.crt` certificate file with base64:
  ```
  cat merged-ca.crt | base64
  ```

8. Replace the old certificate in the Istio Secret. Edit the Secret and replace the `ca.crt` value with the `merged-ca.crt` base64-encoded certificate. Run:
  ```
  kubectl -n istio-system edit secret application-connector-ca-certs
  ```

> 9. If you experiance any issues with new certificates not being trusted, restart Istio Ingress Gateway pods. Run:

Reviewer: ```suggestion
9. If you experience any issues with new certificates not being trusted, restart Istio Ingress Gateway Pods. Run:
```

  ```
  kubectl -n istio-system delete po -l app=istio-ingressgateway
  ```

> > **Note:** To maintain 100% avalibility you can scale up the deployment, restart the old pod and scale the deployment back down.

Reviewer: ```suggestion
> **NOTE:** To maintain 100% availability, you can scale up the deployment, restart the old Pod, and scale the deployment back down.
```

10. Renew the certificates in a runtime. To do that, create a CertificateRequest CR in the runtime in which you want to renew the certificates. Alternatively, wait for the certificates to expire in a given runtime. The system renews the certificates automatically using the information stored in the Secrets you updated.

11. After the certificates are renewed in a runtime, remove the `application-connector-ca-certs` Secret entry which contains the old certificate. First, encode the `new-ca.crt` file with base64:
  ```
  cat new-ca.crt | base64
  ```

12. Edit the Secret and replace the `ca.crt` value with the `new-ca.crt` base64-encoded certificate. Run:
  ```
  kubectl -n istio-system edit secret application-connector-ca-certs
  ```

> 13. Again, for changes to take effect, you may need to restart the pods of Istio Ingess Gateway. Run: 

Reviewer: ```suggestion
13. For changes to take effect, you may need to restart the Pods of Istio Ingress Gateway. Run: 
```

  kubectl -n kyma-integration edit secret connector-service-app-ca
  ```

5. Replace the old certificate in the Istio Secret. Edit the Secret and replace the `ca.crt` value with the new base64-encoded certificate. Run:
  ```
  kubectl -n istio-system edit secret application-connector-ca-certs
  ```

> 6. If you experiance any issues with new certificates not being trusted, restart Istio Ingress Gateway pods. Run:

Reviewer: ```suggestion
6. If you experience any issues with new certificates not being trusted, restart Istio Ingress Gateway Pods. Run:
```

  kubectl -n kyma-integration edit secret connector-service-app-ca
  ```

5. Replace the old certificate in the Istio Secret. Edit the Secret and replace the `ca.crt` value with the new base64-encoded certificate. Run:
  ```
  kubectl -n istio-system edit secret application-connector-ca-certs
  ```

> 6. If you experiance any issues with new certificates not being trusted, restart Istio Ingress Gateway pods. Run:

Reviewer: ```suggestion
6. If you experience any issues with new certificates not being trusted, restart Istio Ingress Gateway pods. Run:
```
